### PR TITLE
Update schema builder

### PR DESCRIPTION
**Pull request by caioflores**
Updating schema builder to reflect our production environment.

- change from class to functions.
- create schema file to back and front.
- remove react getSchema.

### CODE REVIEW

File: api/.env.sample [hunk 1]
```diff
@@ -6,4 +6,6 @@ PG_DATABASE=challenge
 PG_HOST=
 PG_USERNAME=
 PG_PASSWORD=
-PG_PORT=5432
\ No newline at end of file
+PG_PORT=5432
```
Comment (DaniloOliveira28) [inline api/.env.sample]: remove the port info

File: api/src/graphql/schemaBuilder.js [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+const path = module.require('path');
+const { writeFileSync, copyFile } = module.require('fs');
+const { fileLoader, mergeTypes, mergeResolvers } = module.require(
+  'merge-graphql-schemas'
+);
+
+const getPaths = () => {
+  return {
+    typePath: path.join(__dirname, './schema/**/types/*.graphql'),
+    // enumPath: path.join(__dirname, './schema/enums/types'),
```
Comment (DaniloOliveira28) [inline api/src/graphql/schemaBuilder.js]: we are not using enum types?

File: react/.gitignore [hunk 3]
```diff
@@ -11,12 +11,12 @@
 # production
 /build
 
-# misc
-.DS_Store
```
Comment (DaniloOliveira28) [inline react/.gitignore]: The DS_Store file need to remain on gitignore. This is a system file from Mac that are created automatically by the operating system. In order to keep the code cleaned, we need to ignore this kind of file.

File: api/.env.sample (hunk 1 above)
Comment (caioflores) [reply api/.env.sample]: @DaniloOliveira28 it's just a default number for postres connections as example.

File: api/src/graphql/schemaBuilder.js (hunk 2 above)
Comment (caioflores) [reply api/src/graphql/schemaBuilder.js]: Not yet, but we can create some so we can remove the comments.

**Pull request merged**